studiopress/pattern-manager
# Don't allow transforming a 'PM Pattern Block' to any Core block

* Don't allow transforming a 'PM Pattern Block' to any Core block 
* Though 3rd-party blocks can still [allow any](https://github.com/ndiego/gutenberg/blob/05f568f30534d2bfd5bbd050994a19249e1f912f/packages/block-library/src/columns/transforms.js#L16) block to transform to them

# Before

You could transform the 'PM Pattern Block' into 'Columns' or 'Group' blocks, which meant that the 'PM Pattern Block' would be in their `innerBlocks`. 

This might cause styling problems, and weird bug reports.

We can always allow transforming later.

<img width="766" alt="Screenshot 2023-04-20 at 2 11 17 PM" src="https://user-images.githubusercontent.com/4063887/233477672-c537d0d9-31db-43dc-99b8-9437f90c7cc7.png">

# After

<img width="884" alt="Screenshot 2023-04-20 at 2 09 34 PM" src="https://user-images.githubusercontent.com/4063887/233477684-bd2dd337-8ee2-4e29-a3ce-3f8160b5876a.png">

### How to test
1. `cd wp-modules/editor && npm i && cd ../../ && npm run dev`
2. Create a new pattern
3. Add a 'PM Pattern Block'
4. Hover over the icon
5. Expected: No transforms show:

<img width="884" alt="Screenshot 2023-04-20 at 2 09 34 PM" src="https://user-images.githubusercontent.com/4063887/233478384-91a3ef35-2013-4c6f-90b3-854bee11bdcd.png">


### Documentation
No documentation required.
